src/Modules/Workflows/Controllers/WorkflowsList.php

```diff
@@ -171,8 +171,8 @@ public function enqueueScriptsList($hook)
             return;
         }
 
-        global $post_type;
-        if (Module::POST_TYPE_WORKFLOW !== $post_type) {
+        global $postType;
```

Comment:
@ojopaul this is a native variable from WP. Renaming it will break it in the code.

Reply:
I know, it can be a little confuse sometimes

Reply:
Oh, true. Didn't know how I missed that